File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main()
{
    int height, i, j;

    //get integer
    do
    {
        height = get_int("Height: ");
    } while(height < 0 || height > 23);

    //draw pyramid
    for(i = 1; i <= height; i++)
    {
        //left half
        for(j = height - i; j > 0; j--)
            printf(" ");
        for(j = i; j > 0; j--)
            printf("#");
        //seperation
        printf("  ");
        //right half
        for(j = i; j > 0; j--)
            printf("#");
        //new line
        printf("\n");
    }

    return 0;
}

File: pset6/mario.py
from cs50 import get_int

# get int
while True:
    height = get_int("Height: ")
    if height >= 0 and height <= 23:
        break

# print pyramid
for i in range(height):
    print(" " * (height - i - 1), end="")
    print("#" * (i + 1), end="")
    print(" " * 2, end="")
    print("#" * (i + 1))